src/main/java/uk/gov/justice/digital/exceptions/DomainExecutorException.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.exceptions;
+
+public class DomainExecutorException extends Throwable {
+
+    private static final long serialVersionUID = 1L;
+    public DomainExecutorException(String errorMessage) {
```

Comment:
Provide an overload that takes an exception too since this will provide useful info when logged.